L1Trigger/TrackFindingTracklet/interface/Settings.h

```diff
@@ -882,14 +885,15 @@ namespace trklet {
     // This is a temporary fix for compatibilty with HLS. We will need to implement multiple match
     // printing in emulator eventually, possibly after CMSSW-integration inspired rewrites
     // Use false when generating HLS files, use true when doing full hybrid tracking
-    bool doMultipleMatches_{true};
+    bool doMultipleMatches_{false};
 
     // if true, run a dummy fit, producing TTracks directly from output of tracklet pattern reco stage
     bool fakefit_{false};
 
     unsigned int nHelixPar_{4};  // 4 or 5 param helix fit
     bool extended_{false};       // turn on displaced tracking
     bool combined_{false};       // use combined TP (TE+TC) and MP (PR+ME+MC) configuration
+    bool reduced_{false};        // use reduced (Summer Chain) config
```

Comment:
Please update the doc https://github.com/cms-L1TK/firmware-hls/blob/master/README.md so that the section on running the emulation to match the FW includes also instructions on how to set it up for the summer chain.